source/simulator/assemblers/interface.cc

```diff
@@ -450,21 +450,21 @@ namespace aspect
                         finite_element.dofs_per_cell),
           local_matrices_int_ext ((field_is_discontinuous
                                    ?
-                                   GeometryInfo<dim>::max_children_per_face * GeometryInfo<dim>::faces_per_cell
+                                   finite_element.reference_cell().n_faces() * GeometryInfo<dim>::max_children_per_face
```

Comment:
The remaining use of `max_children_per_face` is something I will have to fix in a follow-up. But the replacement I make is easy enough.